mkmsdk/MKMClient.py

```diff
@@ -0,0 +1,17 @@
+from oauthlib.oauth1 import Client
+
+
+class MKMClient(Client):
+    def get_oauth_params(self, request):
+        params = super(MKMClient, self).get_oauth_params(request)
+
+        oauthParamExist = False
```

Comment:
It's a bad behavior using Python keywords as a variable name (in this case ``tuple``). Change it in something else.